### DISCUSSION TITLE
`brew search --desc --eval-all <keyword>` returns nothing

### DISCUSSION THREAD

**Discussion by ElSamhaa** [Homebrew/discussions#4561] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.0.21
ORIGIN: https://github.com/Homebrew/brew
HEAD: 9d0b7361d1a197949dbb47b761405075f83983b7
Last commit: 3 days ago
Core tap JSON: 08 Jun 14:44 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.LIENlgY7Yp/org.xquartz:0
HOMEBREW_EDITOR: nvim
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.3 build 1403
Git: 2.39.2 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.88.1 => /usr/bin/curl
macOS: 13.4-arm64
CLT: 14.3.1.0.1.1683849156
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Whenever I look for a package with a specific keyword in its description/title, I always get an empty result. I've tried many keywords, with the same behavior.

**Comment by CatGreen90** [comment]
It only works with tapped homebrew/core and homebrew/cask for now.

**Reply by ElSamhaa** [reply]
Would the core taps be small enough to not include common keywords in their descriptions?! not a single keyword I tried returned any results?!

<img width="879" alt="image" src="https://github.com/Homebrew/discussions/assets/34912861/b8c90967-82f5-433f-87c6-a31e58645e56">

**Comment by neuhaus** [comment]
I just tried it and it works for me with bash and also with "rss": 

`brew search --desc --eval-all rss`

**Reply by ElSamhaa** [reply, marked as answer]
Yes, it works with me now. Not sure what did it! Maybe a recent update!
